WireAPI/Package.resolved

```diff
@@ -1,6 +1,114 @@
 {
-  "originHash" : "85c56406d543233ecf07c94cba133b478cf3a4a9c438ef2bd018fc13af6cc020",
+  "originHash" : "5cff63afb9c6191f5ba18985bb16fcf342a908e5ed7447eb31f5df612cec0087",
   "pins" : [
+    {
+      "identity" : "aexml",
+      "kind" : "remoteSourceControl",
+      "location" : "https://github.com/tadija/AEXML.git",
+      "state" : {
+        "revision" : "38f7d00b23ecd891e1ee656fa6aeebd6ba04ecc3",
+        "version" : "4.6.1"
+      }
+    },
+    {
+      "identity" : "commander",
+      "kind" : "remoteSourceControl",
+      "location" : "https://github.com/kylef/Commander.git",
+      "state" : {
+        "revision" : "4b6133c3071d521489a80c38fb92d7983f19d438",
+        "version" : "0.9.1"
+      }
+    },
+    {
+      "identity" : "pathkit",
+      "kind" : "remoteSourceControl",
+      "location" : "https://github.com/kylef/PathKit.git",
+      "state" : {
+        "revision" : "3bfd2737b700b9a36565a8c94f4ad2b050a5e574",
+        "version" : "1.0.1"
+      }
+    },
+    {
+      "identity" : "sourcery",
+      "kind" : "remoteSourceControl",
+      "location" : "https://github.com/krzysztofzablocki/Sourcery.git",
+      "state" : {
+        "revision" : "3b84a205f25c58d03369e3775703c7c7ee115a77",
+        "version" : "2.2.4"
+      }
+    },
```

Comment:
This still contains all sub-dependencies, is that right?

Reply:
It seems like the resolved file in the workspace is used rather than this local file, I've deleted it and everything runs fine and a new resolved file wasn't regenerated.